VaticanBAV/et/BAVet57.xml

```diff
@@ -56,7 +56,8 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                            oration after the reading of the gospel (<q xml:lang="gez">ስብሐት፡ ለእግዚአብሔር፡ በሰማያት፡ ወሰላም፡ በምድር፡ ሥምረቱ፡ ለሰብእ፨ በትንብልናሃ፡ ለእግዝእተ፡ ኵልነ፡ ቅድስት፡ 
                               ማርያም፡ ወላዲተ፡ አምላክ፡ ጸግወነ፡ እግዚኦ፡ ስርየተ፡ ኃጣውኢነ፡ ንሰግድ፡ ለከ፡ ኦክርስቶስ፡ ምስለ፡ አቡከ፡ ኄር፡ ወመንፈስ፡ ቅዱስ፡</q>), a prayer entitled 
```

Comment:
Thank you! Ideally, this list should be transformed in 
a list of `<msItem xml:id="ms_i1.2.1">` etc, each with its own `title `and where provided `incipit` but I agree this is a lot of work, so thank you for inserting the reference so it is not lost